pinot-query-runtime/src/test/java/org/apache/pinot/query/service/QueryServerTest.java

```diff
@@ -143,10 +138,9 @@ public void testWorkerAcceptsWorkerRequestCorrect(String sql)
     }
   }
 
-  private static boolean isMetadataMapsEqual(StageMetadata left, StageMetadata right) {
-    return left.getServerInstances().equals(right.getServerInstances())
-        && left.getServerInstanceToSegmentsMap().equals(right.getServerInstanceToSegmentsMap())
-        && left.getScannedTables().equals(right.getScannedTables());
+  private boolean isMetadataMapsEqual(int stageId, DispatchablePlanMetadata stageMetadata,
+      Map<Integer, WorkerMetadata> metadataMap) {
+    return true;
```

Comment:
actually take that back. i can directly compare distributed plan worker metadata with query plan one instead of having to compare dispatchable plan metadata.